src/main/java/Pesel.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+public class Pesel {
+
+    static public void main(String[] args) throws IllegalLengthException, WrongTypeOfDataException {
+
+        Scanner scanner = new Scanner(System.in);
+        String pesel = scanner.nextLine();
+        char[] peselChar = pesel.toCharArray();
+
+        if (pesel.length() != 11) throw new IllegalLengthException(pesel.length());
+        for (int i = 0; i <= peselChar.length - 1; i++ ) {
+            if (!Character.isDigit(peselChar[i])) throw new WrongTypeOfDataException(peselChar[i]);
+        }
+    }
+
+    private static class IllegalLengthException extends Exception {
+
+        private int peselLenght;
+        private IllegalLengthException (int peselLenght){
+            this.peselLenght = peselLenght;
```

Comment:
czy to pole peselLength jest nam do czegoś potrzebne tutaj? Wydaje mi się, że jak to usuniemy to dalej będzie wszystko działało tak samo ;)

Reply:
Usunięte